features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/crypto/identity/IdentityChangeEvent.kt

```diff
@@ -10,5 +10,6 @@ package io.element.android.features.messages.impl.crypto.identity
 import io.element.android.libraries.matrix.api.core.UserId
 
 sealed interface IdentityChangeEvent {
-    data class Submit(val userId: UserId) : IdentityChangeEvent
+    data class PinViolation(val userId: UserId) : IdentityChangeEvent
```

Comment:
Should be renamed to something more related to the action that will be performed. I propose `PinUserIdentity`.